builder/proxmox/common/config.go

```diff
@@ -394,12 +394,12 @@ type rng0Config struct {
 
 // - `vga` (object) - The graphics adapter to use. Example:
```

Comment:
This change looks like it could be mitigated by moving the `-` away from the docs themselves to the inclusion site. I presume `fmt` won't try to align everything based on that if we do so.

Reply:
And while `fmt` does indent correctly, this borks the output.
I suggest we remove the bullet-points in the go docs. Since the parts of the docs that use this structure are already pointed at through anchors, this should be good enough.